### ISSUE TITLE
Reproduce

### ISSUE THREAD

**Issue by zed-2018** [cardwing/Codes-for-autonomous-driving-tasks#1]
Hi,

Your approach to mixed decision making is very inspiring to me. But maybe my level is limited, it seems that it is very difficult to reproduce the results of the paper according to the existing code on the Udacity and comma-ai datasets. Is there any other settings?

**Comment by cardwing** [comment]
Thanks for your interest. I will clean the code and put detailed instructions here soon.

**Comment by zed-2018** [comment]
Hi,
Sorry to bother you again. If you have time, can you release the model that meets the test results of the three datasets in the paper? I want to do some decision-making work on the basis of your feature extraction. Thank you very much!

**Comment by cardwing** [comment]
@zhangende, I will release the trained models soon after the AAAI conference and will inform you once the models are prepared. Sorry about the poor-writing codes.

**Comment by zed-2018** [comment]
Happy New Year,
I did some driving decisions on the BDD100K dataset and improved on the basis of Xu et al. 2017. I hope to refer to your new results, but I can't reproduce the results of FM-Net 85.03% accuracy in your paper. Sincerely hope that you have time to update this trained model. Thank you very much!

**Comment by cardwing** [comment]
Happy New Year,
I will release the trained model soon. If you are anxious, you can try to reproduce our FM-Net by following these steps:
1. modify the network in Xu et al, i.e., from 2D network to 3D ResNet-50 model
2. modify the dataloader file, i.e., from 1 image to 10 images
3. modify the loss function
4. run plain 3D ResNet-50 model and then perform feature mimicking

Edit:
If you have any questions in reproducing the result, feel free to contact me.

**Comment by neel04** [comment]
@cardwing Thanks for such an amazing repo! I had a question - how did you process the BDD100K dataset to extract the appropriate steering angles from it? I'm totally lost on how I can accomplish that 😞
